opentelemetry-appender-log/src/lib.rs

```diff
@@ -170,11 +169,12 @@ where
     L: Logger + Send + Sync,
 {
     pub fn new(provider: &P) -> Self {
+        let scope = InstrumentationScope::builder("opentelemetry-log-appender")
```

Comment:
This scope name should be `opentelemetry-appender-log` , to be consistent with [tracing appender](https://github.com/open-telemetry/opentelemetry-rust/blob/8bd529a6d629aff7482b875cfc39275a8a71eaeb/opentelemetry-appender-tracing/src/layer.rs#L13), so as to follow the convention of `opentelemetry-appender-<name>`. 
But this is not related to PR, I will raise a separate PR for this.